ui/src/overridden/backoffice/Items/ItemCirculationShelf.js

```diff
@@ -1,26 +1,20 @@
+import { isEmpty } from "lodash";
 import React from "react";
-import { Grid, Embed, Button, Icon } from "semantic-ui-react";
+import { Grid, Icon } from "semantic-ui-react";
 import PropTypes from "prop-types";
-import { shelfLink } from "../../utils";
 
 export const ItemCirculationShelf = ({ metadata }) => {
+  let callNumber = {};
+  if (!isEmpty(metadata.identifiers)) {
+    callNumber = metadata.identifiers.find(
+      (identifier) => identifier.scheme === "CALL_NUMBER"
```

Comment:
I might have missed this in other PR, but what is the id scheme `CALL_NUMBER`?

Reply:
it is the old shelf "identifier" which we were asked to preserve and store as call number, since we now have the actual self numbers